docs/contributing.rst

```diff
@@ -227,6 +227,7 @@ Releasing the theme
 
 To release a new version of the theme, core team will take the following steps:
 
+#. Install the required depedencies with ``pip install bump2version wheel twine``.
```

Comment:
We already have these defined in the package configuration as well:

https://github.com/readthedocs/sphinx_rtd_theme/blob/68a89bb07e71fbe222718f680d581dfcd0359769/setup.cfg#L55-L59

Twine should be there though.

Reply:
Done in e09da77